test/e2e-tests.sh

```diff
@@ -33,6 +33,22 @@ function knative_setup() {
   install_knative_serving
 }
 
+function wait_for_leader_controller() {
+  echo -n "Waiting for a leader Controller"
+  for i in {1..150}; do  # timeout after 5 minutes
```

Comment:
can we use `seq` since it's more universal?

Reply:
I could do that but other "wait" functions in library.sh use just this `{1..150}`. Not seeing any benefit in changing that, is there any immediate?

Reply:
🤷 
Not that important. Portability question :-)

Reply:
Hmm. Brace expansion is built in Bash (and this script is supposed to run in Bash: it has `#!/usr/bin/env bash`) . The `seq` is an external utility. You meant portability outside of Bash? perhaps I'd still keep it as it is :)